worldedit-forge/src/main/java/com/sk89q/worldedit/forge/ForgeWorldEdit.java

```diff
@@ -293,6 +295,14 @@ public void onCommandEvent(CommandEvent event) throws CommandSyntaxException {
         ));
     }
 
+    @SubscribeEvent
+    public void onPlayerLogOut(PlayerEvent.PlayerLoggedOutEvent event) {
+        if (event.getPlayer() instanceof ServerPlayerEntity) {
```

Comment:
every day I pray for mojang to move to higher Java versions so we may `instanceof ServerPlayerEntity serverPlayer`.

Reply:
soon:tm: